pkg/datapath/linux/node.go

```diff
@@ -1636,21 +1630,29 @@ func (n *linuxNodeHandler) replaceNodeIPSecOutRoute(ip *net.IPNet) error {
 	return nil
 }
 
-// replaceNodeExternalIPSecOutRoute replace the out IPSec route in the host
-// routing table with the new route. If no route exists the route is installed
-// on the host. The caller must ensure that the CIDR passed in must be non-nil.
-func (n *linuxNodeHandler) replaceNodeExternalIPSecOutRoute(ip *net.IPNet) error {
-	var errs error
+func isValidIP(cfg datapath.LocalNodeConfiguration, ip *net.IPNet) bool {
```

Comment:
Please add godocs for this function.

Reply:
I am not sure this function warrants godocs. It is private and fairly trivial for readers to assert what it does?